### PR TITLE
Automatic generate advanced params files

### PR DESCRIPTION
**Pull request by laysauchoa**
**Motivation** 
We have it used the aiven/scripts to automate documentation. This is a next step where we can run that time to time and keep our documentation update for those automated docs. 

I have decided to add automatic docs in one single workflow to avoid PR noise. However, I can split in two workflows if you think is better (cloud and service advanced params). The PR does not automatically merges, so there will be a manual review to check it out.

**Frequency:**
Not sure how do you feel about the frequency here. I think once per week is okay. But open to suggestions.

**Example:**
Find an example of the automation here: https://github.com/aiven/devportal/pull/952

Fixes: https://github.com/aiven/devportal/issues/444

### CODE REVIEW

**Comment by lornajane** [comment]
Nice! AND our python code benefitted from you looking at it, I knew it was a mess and this is much better :)

If we can, I think it would be good to have separate PRs for each of the `make` commands, so that if things change in multiple products and one can't be merged, the other one isn't blocked - and they might need different reviewers if we need to get a second opinion from someone who has worked on a specific area.

File: Makefile
Comment (mhoffm-aiven) [inline Makefile:87]: I wonder if this is really needed, we could after all just use make and use the other commands as dependencies

Comment (laysauchoa) [reply Makefile]: not needed, fixed

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Comment by laysauchoa** [comment]
Hi @lornajane would be possible to review this PR before I merge it? Also, we need to be aware that the individual PR will be open on Tuesday for each service included. 

Obs: Although, we can still fetch in the document information about elasticsearch, I've not added a workflow for this service as we no longer support it.

**Review by lornajane — APPROVED** [review]
Can we give each job a separate name? It's not a showstopper, and I'm approving and merging anyway but if you could follow up and edit the names, it would help to understand in the logs if one fails.

Thanks for setting this up!

**Pull request merged**
